### DISCUSSION TITLE
What exactly is happening with Google Drive pro(beta), sync is happening good, but data inside the md file is getting deleted

### DISCUSSION THREAD

**Discussion by shanmukhuggini** [remotely-save/remotely-save#812] [Q&A]
Hi,

I am using my vault in two windows laptop, one ipad and one andriod phone. the one drive(free) feature is extremely great. handles all kinds of ways of me syncing and the data of the newly edited or latest one will be present across local and remote.

Today i moved to google drive and synced the whole fours devices after giving auth access.
when i added new pdf file and some .md files in 1 windows laptop and synced it the synced is done and to bring those files and some lines that i wrote in the md into my ipad.

the sync was done , files are downloaded but the data is gone. (its empty apart from my obsidian propeerties) all data is gone (thank god i have an backup)

But what exactly is happening, i kept bidirectional and newest/latest one survices and still all the data is getting erased.

debugging:
1. wrote someting in windows and synced later run the sync in ipad => the data gone.
2. wrote something in ipad and synced later run the sync in windows => the data gone.

unlike onedrive(free) feature where i dont follow any standard steps to sync my data across the devices and remote(onedrive colud)
is there any steps i need to follow in order to make sure the data at what ever device i make changes relfect to another data when i sync properly.

i can use onedrive but the main device i use windows (company laptop) is throwing error 403 forbidden, so that's why i am using google drive feature(which is working fine for commands like create only) but data is getting lost for commands like update, edits.
even i delete files in windows and it is not deleeting in my ipad after successful sync to remote.

let me knoow if you need anything more.

**Comment by shanmukhuggini** [comment]
Also why do i need to give the auth for the google drive for every time i open the obsidian vault.

**Comment by fyears** [comment]
that sounds strange… do you enable smart merge feature?

**Comment by dougkeiller** [comment]
Similar issue for me. 

I just created bug 882 https://github.com/remotely-save/remotely-save/issues/882

**Comment by dougkeiller** [comment]
The reason I'm using Google Drive is because the new Gemini LLM allows me to "QA" my docs, so I can "talk" with my Obisidian docs in Google Drive.
